twilight-http/src/routing.rs

```diff
@@ -1790,12 +1794,12 @@ impl Display for Route<'_> {
                 Display::fmt(guild_id, f)?;
                 f.write_str("/commands")?;
 
-                if let Some(with_localizations) = with_localizations {
-                    f.write_str("?with_localizations=")?;
-                    Display::fmt(with_localizations, f)?;
-                }
+                let query = QueryString::new().with_opt_value(
+                    "with_localizations",
+                    with_localizations.map(|x| x.to_string()),
```

Comment:
Copy is necessary here because into_iter moves non-Copy values which is invalid here without a clone.

Reply:
couldnt we replace `Copy` with `Clone`?

Reply:
We could be there’s no benefit in doing so. We only need readonly access so passing in a &vec (which implements copy) is reasonable. Copying is shallow and cheap whereas cloning is a deep clone of the entire iterable.

Reply:
even if we change `Copy` to `Clone`, if the type already implements `Copy` it will work the same way, `Copy` is simply a marker

when we do this we just allow more types to be passed, its up to the user to follow the general rule of thumb and pass references whenever possible

Reply:
Okay, I can change it as well.